services/user.py

```diff
@@ -0,0 +1,50 @@
+from django.contrib.auth import get_user_model
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> get_user_model():
+    user = get_user_model().objects.create_user(
+        username=username, password=password
+    )
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> get_user_model():
+    return get_user_model().objects.get(pk=user_id)
```

Comment:
Consider handling the `ObjectDoesNotExist` exception in `get_user` to avoid unhandled exceptions if a user with the specified ID does not exist. This can be done by using a try-except block or `get_object_or_404` if applicable.